dojo/tools/aws_prowler_v3/parser.py

```diff
@@ -0,0 +1,104 @@
+            if status.upper() != 'FAIL':
+                continue
+
+            account = deserialized.get('AccountId')
+            region = deserialized.get('Region')
+            provider = deserialized.get('Provider')
+            compliance = str(deserialized.get('Compliance'))
+            result_extended = deserialized.get('StatusExtended')
+            general_description = deserialized.get('Description')
+            asff_compliance_type = " / ".join(deserialized.get('CheckType'))
+            severity = deserialized.get('Severity').capitalize()
+            aws_service_name = deserialized.get('ServiceName')
+            impact = deserialized.get('Risk')
+            mitigation = deserialized.get('Remediation').get('Recommendation').get("Text")
+            mitigation = str(mitigation) + "\n" + str(deserialized.get('Remediation').get('Code'))
+            documentation = deserialized.get('Remediation').get('Recommendation').get("Url")
+            documentation = str(documentation) + "\n" + str(deserialized.get('RelatedUrl'))
+            security_domain = deserialized.get('ResourceType')
+            timestamp = deserialized.get('AssessmentStartTime')
+            resource_arn = deserialized.get('ResourceArn')
+            account_id = deserialized.get('AccountId')
+            resource_id = deserialized.get('ResourceId')
+            if not resource_arn or resource_arn == "":
+                component_name = str(provider) + "-" + str(account_id) + "-" + str(region) + "-" + str(resource_id)
+            else:
+                component_name = resource_arn
+            unique_id_from_tool = deserialized.get('FindingUniqueId')
+
+            description = "**Issue:** " + str(result_extended) + \
+                "\n**Description:** " + str(general_description) + \
+                "\n**AWS Account:** " + str(account) + " | **Region:** " + str(region) + \
+                "\n**Compliance:** " + str(compliance) + \
+                "\n**AWS Service:** " + str(aws_service_name) + \
+                "\n**ASFF Compliance Type:** " + str(asff_compliance_type) + \
+                "\n**Security Domain:** " + str(security_domain)
+
+            # improving key to get duplicates
+            dupe_key = hashlib.sha256(
+                (severity + '|' + region + '|' + result_extended).encode('utf-8')).hexdigest()
+            if dupe_key in dupes:
+                find = dupes[dupe_key]
+                if description is not None:
+                    find.description += description + "\n\n"
+                find.nb_occurences += 1
+            else:
+                find = Finding(
+                    title=textwrap.shorten(result_extended, 150),
+                    cwe=1032,  # Security Configuration Weaknesses, would like to fine tune
+                    test=test,
+                    description=description,
+                    component_name=component_name,
+                    unique_id_from_tool=unique_id_from_tool,
```

Comment:
Use this to update the deduplication settings
https://documentation.defectdojo.com/contributing/how-to-write-a-parser/#deduplication-algorithm